src/main/asciidoc/serializers.adoc

```diff
@@ -0,0 +1,267 @@
+ifndef::sourcedir[]
+:sourcedir: ../..
+endif::sourcedir[]
+
+= Custom Serializers
+
+The Ehcache 3 documentation at http://www.ehcache.org/documentation/3.0/serializers-copiers.html#serializers[Serializers]
+gives you an overview of how to use custom serializers with a cache.
+The section on http://www.ehcache.org/documentation/3.0/serializers-copiers.html#persistent-vs-transient-caches[Persistent and Transient Serializers]
+briefly covers the serializer contracts that must be honored while writing custom serializers to be used with 
+persistent/transient caches.
+
+This document explains how you can write a transient/persistent custom serializer that works with Ehcache.
+Here we discuss the significance of transient serializers and persistent serializers in detail through a practical example
+
+== Serializer types
+
+As indicated in the Ehcache documentation, serializers require a single argument constructor or a double-argument 
+constructor or both based on the type of the cache they are used in.
```

Comment:
... on the type of cache ...

I suggest dropping the "the" in front of cache here.